projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.html

```diff
@@ -139,9 +141,9 @@
         </div>
       </div>
       <appointments-overview [node]="node"></appointments-overview>
-      <div *ngIf="hasErrors()">
-        <div *ngFor="let e of errors()">
-          <span class="text-danger">{{ e }}</span>
+      <div  *ngIf="hasErrors()">
```

Comment:
Any reason for the usage of double spaces?

Reply:
Should be sorted now